mapit/management/commands/mapit_import.py

```diff
@@ -215,16 +215,15 @@ def verbose(*args):
                 name = override_name
             else:
                 try:
-                    name = feat[name_field].value
-                except:
-                    choices = ', '.join(layer.fields)
-                    raise CommandError(
-                        "Could not find name using name field '%s' - should it be something else? "
-                        "It will be one of these: %s. Specify which with --name_field" % (name_field, choices))
-                try:
+                    name = feat.get(name_field)
```

Comment:
I think Feature's `get()` throws an `OGRIndexError` exception if name_field isn't present, unlike the normal pythonic `get()`, so the `None` check won't be reached.